### PR TITLE
[lrt] fix the broken extractor

### PR DESCRIPTION
**Pull request by GiedriusS**
The website has moved to using m3u8 so changes are needed as a result. The "real" format is still mp4. Also, makes the test pass again with no changes to it

### CODE REVIEW

**Comment by GiedriusS** [comment]
@naglis ping

**Comment by GiedriusS** [comment]
Ping

**Comment by GiedriusS** [comment]
Ping

File: youtube_dl/extractor/lrt.py [hunk 1]
```diff
@@ -37,26 +33,11 @@ def _real_extract(self, url):
...
-                    'url': data['streamer'],
-                    'play_path': 'mp4:%s' % data['file'],
-                    'preference': -1,
-                    'rtmp_real_time': True,
-                })
-            else:
-                formats.extend(
-                    self._extract_m3u8_formats(data['file'], video_id, 'mp4'))
+        link = self._search_regex(r'file: "(.*)" \+ location\.hash\.substring\(1\)', webpage, 'link to m3u8')
+        link += video_id
```
Comment (jaimeMF) [inline youtube_dl/extractor/lrt.py]: The test video (http://www.lrt.lt/mediateka/irasas/54391/) seems to work fine without this line, and it doesn't seem to be used by their webpage (`location.hash` is empty), so I'm going to remove it.

**Comment by jaimeMF** [comment]
Merged, thanks! (And sorry for the delay)

**Pull request closed without merging**